### DISCUSSION TITLE
How set the specific directory?

### DISCUSSION THREAD

**Discussion by sohamaa** [Sanster/IOPaint#487] [Q&A] [answered]
How set the specific directory (ex: D: models, instead of the default path) for loading all models and stuff? (Not by One-Click installer).
Every time I want to load my models I have to run --dir (path) to load my models, if I only run the IOpaint run, it starts to download all the models in C: default path again.
How can I make it the default path? It seems there is a config file in the one-click installer but how can I set it manually?

How can I add this? 
![image](https://github.com/Sanster/IOPaint/assets/119129683/b43eb359-74fd-44de-b2b4-1e674f5e85d4)

**Comment by Sanster** [comment, marked as answer]
Using `--model-dir` will set the environment variable `XDG_CACHE_HOME`, which will be used by [diffusers](https://github.com/huggingface/diffusers).  

You can use `iopaint start-web-config  --config-file installer-config.json` to start a gradio page for configuration settings, the configuration will be saved in `installer-config.json` (including model_dir), and then use `iopaint start --config installer-config.json` to start the service.

**Comment by sohamaa** [comment]
It works! Thank you so much!
